### PR TITLE
fix(TaskJobState): fix SQL error on fail

### PR DESCRIPTION
**Pull request by stonebuzz**
Try to fix some SQL error when collect fail

see 354

### CODE REVIEW

**Review by trasher — APPROVED** [review]
LGTM. I'll wait for user confirmation before merging.

File: inc/collect.class.php
Comment (cedric-anne) [inline inc/collect.class.php:863]:
```suggestion
                        $pfTaskjobstate->getFromDB($jobstate['id']);
```

**Review by cedric-anne — APPROVED** [review]

**Pull request merged**
